gubernator/templates/build.html

```diff
@@ -34,6 +34,8 @@
 				% set pod_name = text|parse_pod_name
 				% if pod_name
 					<p>Find <tt>{{pod_name}}</tt> mentions in <a id="{{pod_name|slugify}}" href="/build{{build_dir}}/nodelog?pod={{pod_name}}&junit={{filename|basename}}&wrap=on">log files</a>
+				% else
+					<p>Filter through <a href="/build{{build_dir}}/nodelog?junit={{filename|basename}}&wrap=on">log files</a>
```

Comment:
Can you give an example failure where the build logs are useful but no pod name was detected?